packages/back-end/src/types/Integration.ts

```diff
@@ -181,6 +224,10 @@ export interface SourceIntegrationInterface {
   ): Promise<ExperimentQueryResponses>;
   getSourceProperties(): DataSourceProperties;
   testConnection(): Promise<boolean>;
+  getInformationSchema(
+    datasourceType?: string,
+    projectId?: string
```

Comment:
We don't need either of these arguments to get the information schema.  `projectId` is specific to BigQuery.  And I believe you can already get the data source type directly from the Integration object.